app.js

```diff
@@ -42,211 +49,230 @@ let transporter = nodemailer.createTransport({
-          expiresIn: '30d' // expires in x days
-        });
+                    console.log('..user created');
 
-        res.cookie('token', token, {
-          expires : token.expiresIn,
-          httpOnly : false
-        });
+                    // User has logged in
+                    let token = jwt.sign({ ethAddress: newUser.ethAddress }, process.env.TOKEN_SECRET, {
+                        expiresIn: '30d' // expires in x days
+                    });
 
-        cb(null, newUser);
-      });
-    }
-  });
+                    res.cookie('token', token, {
+                        expires: token.expiresIn,
+                        httpOnly: false
+                    });
+
+                    cb(null, newUser);
+                });
+            }
+        });
 }
 
 
-function authenticateRequest(req, res, cb) {
-  let token = req.body.token || req.query.token || req.headers['x-access-token'] || req.cookies.token;
-  
-  // Check if token exists
-  if (!token) {
-    console.log('User didnt provide token..');
-    res.redirect('/login');
-    return;
-  }
-
-  // Verify it
-  jwt.verify(token, process.env.TOKEN_SECRET, function(err, decoded) {      
-    if (err) {
-      console.log('Invalid login..');
-      res.sendFile(path.join(__dirname + '/public/login'));
-      return;
-    } else {
-
-      // Find the user associated with this email
-      models.User.findOne({ where: {ethAddress: decoded.ethAddress} }).then(user => {
-        if (!user) {
-          // invalid user login
-          console.log('Invalid login email..');
-          return res.redirect('/login');
+const authenticateRequest = (req, res, cb) => {
+    let token = req.body.token || req.query.token || req.headers['x-access-token'] || req.cookies.token;
+
+    // Check if token exists
+    if (!token) {
+        console.log('User didnt provide token..');
+        res.redirect('/login');
+        return;
+    }
+
+    // Verify it
+    jwt.verify(token, process.env.TOKEN_SECRET, function(err, decoded) {
+        if (err) {
+            console.log('Invalid login..');
+            res.sendFile(path.join(__dirname + '/public/login'));
+            return;
         } else {
-          cb(user);
+
+            // Find the user associated with this email
+            models.User.findOne({ where: { ethAddress: decoded.ethAddress } }).then(user => {
+                if (!user) {
+                    // invalid user login
+                    console.log('Invalid login email..');
+                    return res.redirect('/login');
+                } else {
+                    cb(user);
+                }
+            });
         }
-      });
-    }
-  });
+    });
 }
 
 
 //////////////////
 // Static Files //
 //////////////////
 
-app.get('/', function (req, res) {
-  res.sendFile(path.join(__dirname + '/build/index.html'));
+app.get('/', function(req, res) {
+    res.sendFile(path.join(__dirname + '/build/index.html'));
```

Comment:
Did you change all the 2 space indents into 4 spaces? Is that what all this is?

Reply:
Sorry yeah all of these are just spacing changes. Would you like me to restore those?

Reply:
No I think it's fine. @trevorlyman and I are the only other ones that touch this app.js file so it shouldn't cause too many merge conflicts later.